plugins/base/src/main/kotlin/renderers/html/innerTemplating/DefaultTemplateModelFactory.kt

```diff
@@ -91,29 +92,41 @@ class DefaultTemplateModelFactory(val context: DokkaContext) : TemplateModelFact
         get() = URI(this).isAbsolute
 
     private fun Appendable.resourcesForPage(pathToRoot: String, resources: List<String>): Unit =
-        resources.forEach {
+        resources.forEach { resource ->
+            /**
+             * Workaround for resources that have no extension
+             * e.g. `<script src="`https://unpkg.com/kotlin-playground@1`"></script>`
```

Comment:
If this resource is the only reason this workaround is needed, it looks like `https://unpkg.com/kotlin-playground@1/dist/playground.min.js` could be used instead - it only has the major version hardcoded (`@1`), but still redirects to the same page with `@1.28.0`.

Reply:
https://github.com/JetBrains/kotlin-playground or https://www.npmjs.com/package/kotlin-playground  uses `https://unpkg.com/kotlin-playground@1`
Potentially the file path can be changed, but the chance is negligible.

Reply:
Sorry, but I really think this workaround can be easily avoided, and I think it should be, especially given that there are no tests and we have no plans to address it in the near future.

Is there a problem with using `https://unpkg.com/kotlin-playground@1/dist/playground.min.js`? As you said, the chance that the URL/path change is low, and I would honestly be more worried about us using a third party CDN in case of a supply chain attack. 

If you want this to be stable and work for years to come, I think the way forward is to ask the Kotlin Website team to provide a stable url instead, and we won't have to worry about it. In the meantime, it looks like we can just use `https://unpkg.com/kotlin-playground@1/dist/playground.min.js`.